srcpkgs/python3-i3ipc/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-i3ipc'
+pkgname=python3-i3ipc
+version=1.7.1
+revision=1
+archs=noarch
+wrksrc="i3ipc-python-${version}"
+build_style=python3-module
+pycompile_module="python3-i3ipc"
+hostmakedepends="python3-setuptools"
+depends="python3 python3-setuptools"
```

Comment:
I don't think we need `python3-setuptools` in `depends`

Reply:
I wasn't sure. To be removed.

Reply:
Done.